CHANGELOG.md

```diff
@@ -30,7 +30,7 @@ All notable changes to this project will be documented in this file. The format
 ### Frontend
 
 - Added:
-
+  - Adds filtering capability to listings list and implements on public site
```

Comment:
Do you mind adding the link to the PR? There should be other examples in the changelog :)

Reply:
Added.